src/resources/account/Preferences.js

```diff
@@ -0,0 +1,59 @@
+import DeskbookersError from '../../DeskbookersError'
+import Resource from '../Resource'
+
+export default class Preferences extends Resource {
+  constructor (api) {
+    super(api)
+    this.endpoint = 'user/preferences'
+  }
+
+  async list () {
+    const preferences = await this.request({
+      method: 'GET',
+      path: this.endpoint
+    })
+
+    const map = new Map()
+    for (const item in preferences) {
+      map.set(item, preferences[item])
+    }
+
+    return map
+  }
+
+  async retrieve (...keys) {
```

Comment:
Can be confusing behaviour in the following situation:

```js
// Single string
const value = deskbookers.account.preferences.retrieve('key')

// Multiple strings (1)
const values = deskbookers.account.preferences.retrieve('key1', 'key2')

// Multiple strings (2)
const keys = ['key1', 'key2']
const values = deskbookers.account.preferences.retrieve(...keys)

// Multiple strings (3)
const keys = ['key1']
const values = deskbookers.account.preferences.retrieve(...keys)
// Few problems:
// The argument `keys` gets passed from 'far away'
// We expect `keys` to be an array, so we expect the multiple key mode
// But when `keys` is empty or fails
// And when `keys` only contains one string it returns one string instead of an array of strings
```

Maybe we should use the argument as either an array or a string instead of `...keys`